### PR TITLE
Added a payload property to MessageEntity and made sender nullable

### PR DESCRIPTION
**Pull request by awartoft**
The payload property allows for adding meta data to the messages, we also made sender nullable. Doing this allows for combining system  messages without a sender and payload for meta data to do fun stuff in
the frontend.

### CODE REVIEW

File: src/InteractiveSolutions/UserMessage/Entity/MessageEntity.php [hunk 1]
```diff
@@ -78,64 +91,73 @@ public static function readByUser(MessageEntity $instance, ReadByUserEntry $entr
...
+    public function getId(): int
     {
         return $this->id;
     }
 
     /**
      * @return MessageUserInterface
      */
-    public function getSender()
+    public function getSender(): MessageUserInterface
```
Comment (noggan) [inline src/InteractiveSolutions/UserMessage/Entity/MessageEntity.php]: Sender can be null u fool

Comment (noggan) [reply src/InteractiveSolutions/UserMessage/Entity/MessageEntity.php]: `?MessageUserInterface` if php 7.1 otherwise you can't type it.

Comment (awartoft) [reply src/InteractiveSolutions/UserMessage/Entity/MessageEntity.php]: Hmm, do we want to poke the bear and upgrade to 7.1 ? I don't see why not.

Comment (Hotas2k) [reply src/InteractiveSolutions/UserMessage/Entity/MessageEntity.php]: Do it!

**Pull request merged**